File: lab07/total_words.pl
#!/usr/bin/perl -w
$count = 0;
@file = <STDIN>;
foreach $l (@file) {
    $l =~ s/'s/ s/g;
    $l =~ s/[^a-zA-Z]/ /g;
    @line = split/\s+/, $l;
    foreach $w (@line){
	if ($w ne "") {
	    $count++;
	}
    }
}
if ($count > 1) {
    print("$count words\n");
}
else {
    print("$count word\n");
}

File: lab07/total_words.py
#!/usr/bin/python3
import sys
import re
count = 0
for line in sys.stdin:
    line = re.sub(r'[^A-Za-z]',' ',line)
    line = re.sub(r'\'s',' s',line)
    word = re.split(r'\s+',line)
    for x in word:
        if x != '':
            count += 1
print("{} words".format(count))
